website/docs/r/is_ike_policy.html.markdown

```diff
@@ -40,9 +40,21 @@ resource "ibm_is_ike_policy" "example" {
 ## Argument reference
 Review the argument references that you can specify for your resource. 
 
-- `authentication_algorithm` - (Required, String) Enter the algorithm that you want to use to authenticate `IPSec` peers. Available options are `md5`, `sha1`, or `sha256`.
-- `dh_group`  - (Required, Integer) Enter the Diffie-Hellman group that you want to use for the encryption key. Available enumeration type are `2`, `5`, `14`, or `19`.
-- `encryption_algorithm` - (Required, String) Enter the algorithm that you want to use to encrypt data. Available options are: `triple_des`, `aes128`, or `aes256`.
+- `authentication_algorithm` - (Required, String) Enter the algorithm that you want to use to authenticate `IKE` peers. Available options are `md5`, `sha1`,  `sha256`, `sha512`, `sha384`.
+  
+  ~> **Note**
+  The `md5` and `sha1` algorithms have been deprecated and will be removed from November 30 2022
```

Comment:
The `md5` and `sha1` algorithms have been deprecated and support will be removed from November 2022.